docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -8,6 +8,65 @@ ifdef::notBuildingForSite[]
 include::{includedir}/menu.adoc[]
 endif::notBuildingForSite[]
 
+== Getting Ehcache
+
+=== From Maven
+
+Add the following to the <dependencies> section in your pom.xml:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.ehcache</groupId>
+    <artifactId>ehcache</artifactId>
+    <version>3.10.0</version>
+</dependency>
+----
+
+=== From Gradle
+
+Add the following to the dependencies section of your build.gradle:
+
+[source,groovy]
+----
+dependencies {
+  implementation 'org.ehcache:ehcache:3.10.0'
+}
+----
+
+ehcache-xml and ehcache modules requires jaxb dependency. So If you are using jdk-9+ version then you should make jaxb dependency available at runtime.
+
+1. We depend on a version range, Version for jaxb dependency should range between 2.2.0 - 3.0.0
+2. We have upgraded the jaxb api to jakarta version
+
```

Comment:
I see that there were previously some changes made to this section (these few sentences)... and the text is left a bit rough.  Could we please update it to the following?:

```
The modules ehcache-xml and ehcache have a dependency upon JAXB.  If you are using JDK-9 or greater then you need to ensure the JAXB dependency is available at runtime.

Please note:
1. The JAXB dependency declared by Ehcache is with the version range 2.2.0 - 3.0.0.   You will probably want to specify a particular version, in order to get consistent builds of your components.
```